### PR TITLE
First implementation of Map; prepared other files

### PR DESCRIPTION
**Pull request by swirszcz-adrian**

### CODE REVIEW

File: SouvlakMVP/SouvlakMVP.sln
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP.sln:1]: I trust it's correct ;)

File: SouvlakMVP/SouvlakMVP/Solution.cs
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP/Solution.cs:1]: What solution? Program-solution or `Genetic Algorithm`-solution?

File: SouvlakMVP/SouvlakMVP/Program.cs
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP/Program.cs:1]: Hardcoded stuff for MVP debugging here, right?

File: SouvlakMVP/SouvlakMVP/Map.cs [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+            }
+            else
+            {
+                return false;
+            };
+        }
+
+        public override int GetHashCode()
+        {
+            return this.targetIdx.GetHashCode() * 997 + this.distance.GetHashCode();    // just multiply one value by big prime number
```
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP/Map.cs:57]: `997` as an EasterEgg? :p

File: SouvlakMVP/SouvlakMVP/Map.cs
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP/Map.cs:1]: For now, I trust it i correct (will see later :p)

**Review by Kri5t0fK — APPROVED** [review]
For that 1. commit with MVP stuff I trust it's correct ;)

File: SouvlakMVP/SouvlakMVP/Solution.cs
Comment (Kri5t0fK) [inline SouvlakMVP/SouvlakMVP/Solution.cs:1]: `Solution` -> elaborate

**Review by Kri5t0fK — CHANGES_REQUESTED** [review]

File: SouvlakMVP/SouvlakMVP/Program.cs
Comment (swirszcz-adrian) [reply SouvlakMVP/SouvlakMVP/Program.cs:1]: Yup, since net maui has it's own way of accesing files

File: SouvlakMVP/SouvlakMVP/Solution.cs
Comment (swirszcz-adrian) [reply SouvlakMVP/SouvlakMVP/Solution.cs:1]: What solution? I dont see a file like it in newest commit :3

**Review by Kri5t0fK — APPROVED** [review]
Ok, now is good

**Pull request merged**